features/popups/popup-launcher.html

```diff
@@ -0,0 +1,111 @@
+﻿<!DOCTYPE html>
+
+<html lang="en" xmlns="http://www.w3.org/1999/xhtml">
+<head>
+    <meta charset="utf-8" />
+    <title>Popup test launcher</title>
```

Comment:
Please use the template for the top part of the document - https://github.com/duckduckgo/privacy-test-pages/blob/main/TEMPLATES/simple.html . There are couple of small things that template asks you to do:
- link to list of pages ("Home") (for easy navigation)
- description that explains what this page is supposed to test (to be understandable to anyone browsing existing pages)
- title of the page consistent with the name of the page in index (to avoid confusion)